src/main/resources/public/assets/css/kotStyle.css

```diff
@@ -0,0 +1,62 @@
+.activeOrdersColumn {
+	background: #1dab76;
+    height: 50px;
+}
+
+.activeOrders {
+	line-height: 50px;
+    font-size: 1.3em;
+    color: white;
+}
+
+.wrapperColumn {
```

Comment:
Is this the same base theme we are using for the whole web? If so, use the default class instead of redefining.